CMakeLists.txt

```diff
@@ -143,12 +158,16 @@ set_property(
         PUBLIC_HEADER "${API_HEADER_FILES}"
 )
 
+target_compile_definitions(uriparser PUBLIC -DURI_LIBRARY_BUILD)
```

Comment:
This definition is only useful during the building process of `uriparser`, and should not propagate to the CMake module file `uriparser.cmake`. 
Besides, the `-D` prefix is unnecessary according to: https://cmake.org/cmake/help/latest/command/target_compile_definitions.html

```cmake
target_compile_definitions(uriparser PRIVATE URI_LIBRARY_BUILD)
```
may be better.

Reply:
That's a great catch, thank you!